ocrd_segment/classify_formdata_layout.py

```diff
@@ -516,9 +541,6 @@ def extend_mask(masks, match):
         newmax = tf.maximum(roimax, compmax)
         # ignore background (matches everywhere):
         background = tf.identity(complabels[compidx] == 0)
-        # skip if wc > 2 * w or hc > 2 * h:
-        toolarge = tf.reduce_any(compmax - compmin > 2 * (roimax - roimin))
-        # skip if neww > 2 * w or newh > 1.5 * h:
```

Comment:
This removal was by accident I suppose?

```suggestion
        # skip if wc > 2 * w or hc > 2 * h:
        toolarge = tf.reduce_any(compmax - compmin > 2 * (roimax - roimin))
        # skip if neww > 2 * w or newh > 1.5 * h:
```